src/Main.java

```diff
@@ -1,17 +1,89 @@
+import java.time.LocalDate;
+
 public class Main {
+    public static void main(String[] args) {
+        printIsLeapYear(2024);
+        recomendApplicationVersion(0, 2022);
+        System.out.println(calculateDeliveryDays(55));
+    }
 
-    //Task 1
-    public static void calculateLeapYear() {
-        int year = 2024;
-        boolean leapYear = (year >= 1584 && ((year % 400 == 0) || (year % 4 == 0 && year % 100 != 0)));
-        if (leapYear) {
-            System.out.println(year + " год является високосным.");
+        private static void printIsLeapYearResult(int year, boolean yearIsLeap) {
+        if (yearIsLeap) {
+            System.out.println(year + " високосный");
         } else {
-            System.out.println(year + " год не является високосным.");
+            System.out.println(year + " не является високосным");
         }
     }
 
-    public static void main(String[] args) {
-        System.out.println("Hello world!");
+    public static void printIsLeapYear(int year) {
+        boolean yearIsLeap = isLeap(year);
+        printIsLeapYearResult(year, yearIsLeap);
+    }
+
+
+    private static boolean isLeap(int year) {
+        return year % 400 == 0 || year % 4 == 0 && year % 100 != 0;
+    }
+
+    public static void recomendApplicationVersion(int clientOS, int deviceYear) {
+        boolean deviceIsOld = isDeviceOld(deviceYear);
+        System.out.println("Установите ");
+        if (deviceIsOld) {
+            System.out.println("Lite ");
+        }
+        System.out.println("версию для ");
```

Comment:
```suggestion
        System.out.print("версию для ");
```
перенос строки не нужен